ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtilTest.java

```diff
@@ -319,7 +320,74 @@ void penalizeValidatorDecrementsBadActorAndIncrementsWhistleblower() {
     beaconState.setCurrent_shuffling_epoch(Constants.FAR_FUTURE_EPOCH);
     beaconState.setPrevious_shuffling_epoch(Constants.FAR_FUTURE_EPOCH);
     List<Long> latestPenalizedBalances =
-        new ArrayList<>(Arrays.asList(randomLong(), randomLong(), randomLong()));
+        new ArrayList<>(
```

Comment:
I'd suggest using something like https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#nCopies(int,%20T) so this isn't so verbose.